app/ceom/maps/templates/maps/leaderboard.html

```diff
@@ -0,0 +1,81 @@
+{% extends "maps/base.html" %}
+{% block content %}
+
+<h2 class="legend title">Pixels Leaderboard</h2>
+
+<h5 class="text-center"> Welcome to the <strong> CEOM Pixels Categorize Leaderboard</strong>! </h5>
```

Comment:
Typo here ("Pixels Categorize Leaderboard"), also generally I don't use introduction messages like this

I would recommend just removing this line, given that it repeats the information in the line above it
I'll take a look at it whenever it's put into production and I might change my mind